kolibri/plugins/coach/assets/src/views/recent-page/channel-list.vue

```diff
@@ -28,10 +32,44 @@
 
   module.exports = {
     name: 'channelList',
-    $trNameSpace: 'channelList',
+    $trNameSpace: 'coachRecentPageChannelList',
     $trs: {
       channels: 'Channels',
       channelList: 'Channel list',
+      lastActive: 'Last active',
+      daysPassed: 'X day(s) ago',
+      monthsPassed: 'X month(s) ago',
+      timePassed:
+      '{amount, number} {measure, select, ' +
+        'day {' +
+          '{amount, plural,' +
+            'one {day}' +
+            'other {days}' +
+          '}' +
+        '}' +
+        'month {' +
+          '{amount, plural,' +
+            'one {month}' +
+            'other {months}' +
+          '}' +
+        '}' +
+      '} ago',
```

Comment:
@DXCanas 

Unfortunately we can't use multi-line expressions like this for `$tr` strings because it breaks string extraction. Watch out for this message during build:

> Kolibri: The value for $trs "timePassed", is not valid. Make sure it is not an expression. Found in /Users/d/Projects/learning_equality/repos/kolibri/kolibri/plugins/coach/assets/src/views/recent-page/channel-list.vue.